test/flows/student_finance_calculator_flow_test.rb

```diff
@@ -0,0 +1,362 @@
+    context "question: where_will_you_live_while_studying?," do
+      setup do
+        testing_node :where_will_you_live_while_studying?
+        add_responses when_does_your_course_start?: "2020-2021",
+                      what_type_of_student_are_you?: "uk-full-time",
+                      how_much_are_your_tuition_fees_per_year?: "9250"
+      end
+
+      should "render the question" do
+        assert_rendered_question
+      end
+
+      context "next_node" do
+        should "have a next node of whats_your_household_income? for any response" do
+          assert_next_node :whats_your_household_income?, for_response: "at-home"
+        end
+      end
+    end
+
+    context "question: whats_your_household_income?," do
+      setup do
+        testing_node :whats_your_household_income?
+        add_responses when_does_your_course_start?: "2020-2021",
+                      what_type_of_student_are_you?: "uk-full-time",
+                      how_much_are_your_tuition_fees_per_year?: "9250",
+                      where_will_you_live_while_studying?: "at-home"
+      end
+
+      should "render the question" do
+        assert_rendered_question
+      end
+
+      context "next_node" do
+        should "have a next node do_any_of_the_following_apply_uk_full_time_students_only? for any response" do
+          assert_next_node :do_any_of_the_following_apply_uk_full_time_students_only?, for_response: "50,000"
+        end
+      end
+    end
+
+    context "question: do_any_of_the_following_apply_uk_full_time_students_only?," do
+      setup do
+        testing_node :do_any_of_the_following_apply_uk_full_time_students_only?
+        add_responses when_does_your_course_start?: "2020-2021",
+                      what_type_of_student_are_you?: "uk-full-time",
+                      how_much_are_your_tuition_fees_per_year?: "9250",
+                      where_will_you_live_while_studying?: "at-home",
+                      whats_your_household_income?: "50,000"
+      end
+
+      should "render the question" do
+        assert_rendered_question
+      end
+
+      context "next_node" do
+        should "have a next node what_course_are_you_studying? for any response" do
+          assert_next_node :what_course_are_you_studying?, for_response: "children-under-17,dependant-adult,has-disability,low-income"
+        end
+      end
+    end
+
+    context "question: what_course_are_you_studying?," do
+      setup do
+        testing_node :what_course_are_you_studying?
+        add_responses when_does_your_course_start?: "2020-2021",
+                      what_type_of_student_are_you?: "uk-full-time",
+                      how_much_are_your_tuition_fees_per_year?: "9250",
+                      where_will_you_live_while_studying?: "at-home",
+                      whats_your_household_income?: "50,000",
+                      do_any_of_the_following_apply_uk_full_time_students_only?: "children-under-17,dependant-adult,has-disability,low-income"
```

Comment:
it'd be an easier read if we went for a simpler response here (I imagine we just need the first one?)